keypad/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+set(TARGET_NAME keypad)
+add_executable(${TARGET_NAME}
+    keypad.c
+    )
+
+# pull in common dependencies
+target_link_libraries(keypad pico_stdlib)
+
+if (PICO_CYW43_SUPPORTED)
+	target_link_libraries(${TARGET_NAME} pico_cyw43_arch_none)
+endif()
```

Comment:
This doesn't seem needed, as none of your code uses CYW43 functionality?

Reply:
Thanks again! I forgot to remove this as I took the blink code as example!
Fixed in https://github.com/raspberrypi/pico-examples/pull/542/commits/f7874196fac2fea5c58beb68f55499885a857e6c